src/unix/proctitle.c

```diff
@@ -50,7 +50,9 @@ char** uv_setup_args(int argc, char** argv) {
 
   process_title.str = argv[0];
   process_title.len = argv[argc - 1] + strlen(argv[argc - 1]) - argv[0];
+#ifndef __MVS__
   assert(process_title.len + 1 == size);  /* argv memory should be adjacent. */
```

Comment:
For some reason, z lines up the arguments differently (not adjacent)  :-(
Here is unix:
[exe name][null char][arg1][null char][arg2][null char][arg3][null]
Here is zos:
[exe name][data][arg1][data][arg2][data][arg3][data]
where
data = integer (4 bytes) which is the length of the next arg string.

I will insert a new assertion for the zOS case.